### PR TITLE
[OKTA-543635] Reserved attributes in SCIM

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Added info about reserved attributes to SCIM doc
- **Is this PR related to a Monolith release?**  Nope!

### Resolves:

* [OKTA-543635](https://oktainc.atlassian.net/browse/OKTA-543635)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md](https://github.com/okta/okta-developer-docs/blob/7edac01c00404f12c212746707fcebb6c43c6d69/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Ffaqs%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94f77948-fdfc-402d-aa7e-d778d527031a) | 79  | 75  | 76  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.5c6fd899-5bd8-4496-8a85-94f92a3b49eb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ You should also specify your data requirements in the configuration document tha
 
 **Q: How much support is required for filtering results?**
 
-The filtering capabilities in the SCIM protocol are pretty broad, but the common filtering use case with Okta is quite narrow. Okta determines if a newly created user or group exists in your app based on a matching identifier. This means the `eq` (equals) operator is all you really need to support.
+The filtering capabilities in the SCIM protocol are broad, but the common filtering use case with Okta is narrow. Okta determines if a newly created user or group exists in your app based on a matching identifier. This means the `eq` (equals) operator is all you really need to support.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md:79]:
```suggestion
The filtering capabilities in the SCIM protocol are broad, but the common filtering use case with Okta is narrow. Okta determines if a newly created user or group exists in your app based on a matching identifier. This means the `eq` (equals) operator is all you need to support.
```

File: packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md [hunk 2]
```diff
@@ -111,3 +111,7 @@ The [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integ
 **Q: What is the timeout if Okta doesn't receive a response from my SCIM server?**
 
 The timeout is 60 seconds if Okta doesn't receive anything from the socket after any SCIM request is made.
+
+**Q: Are there reserved attributes that I can't use?**
+
+Okta follows the SCIM specifications as defined by [RFC 7642](https://datatracker.ietf.org/doc/html/rfc7642), [RFC 7643](https://datatracker.ietf.org/doc/html/rfc7643), and [RFC 7644](https://datatracker.ietf.org/doc/html/rfc7644), with exceptions listed in the reference guides. All system or reserved attributes follow these RFC documents. For example, `groups` is a reserved and used property on the `User` object defined as part of RFC 7643, so you shouldn't try to create a custom attribute called `groups` in your SCIM server.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md:117]:
```suggestion
Okta follows the SCIM specifications as defined by [RFC 7642](https://datatracker.ietf.org/doc/html/rfc7642), [RFC 7643](https://datatracker.ietf.org/doc/html/rfc7643), and [RFC 7644](https://datatracker.ietf.org/doc/html/rfc7644), with exceptions listed in the reference guides. All system or reserved attributes follow these RFC documents. For example, `groups` is a reserved and used property on the `User` object defined as part of RFC 7643. Therefore, don't create a custom attribute called `groups` in your SCIM server.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md](https://github.com/okta/okta-developer-docs/blob/3884a1a9d21605d25c8cb614847b985e661707a7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Ffaqs%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bf4fbb1e-514e-49cc-b774-2ce22d5033cc) | 79  | 75  | 90  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.80771905-4768-4cd3-a2ef-b5ce2af7f88d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/faqs/index.md](https://github.com/okta/okta-developer-docs/blob/4f2ea45566ba6620732c81ebb128fedd67dbfd80/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Ffaqs%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/71d62427-765a-4c3e-8dda-876068036771) | 79  | 75  | 90  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b397353f-c596-44bf-bff5-61c4ad4a511f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**